README.md

```diff
@@ -226,6 +231,74 @@ plugin.on('deviceDidDisconnect', ({ device }) => console.log(`device with id ${d
 
 ---
 
+### dialPress
+
+**[Stream Deck 6.0.0+ Required]**
+
+Triggered when a user presses or releases the dial on a Stream Deck +.
+
+**Event-Payload**:
+
+```typescript
+event: {
+  row: number | undefined;
+  column: number | undefined;
+  action: string;
+  context: string;
+  device: string;
+  settings: unknown;
+  controller: ControllerType; // String enum, but guaranteed to be "Encoder".
```

Comment:
sounds a bit weird - like it's always "Encoder".
probably fine to just hint the two possible strings?

Reply:
These events are only ever actually sent from an `Encoder`, so that's why I put that comment in there. I'm mostly just trying to avoid users putting in (unnecessary) checks for controller type.

If you still want me to change this to just the members though, I have no objections.

Reply:
right, makes sense. thanks for clarifying